### PR TITLE
[16.0][IMP] rma: search by rma origin and add state filters

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Thanks :+1:

File: rma/views/rma_views.xml [hunk 1]
```diff
@@ -9,8 +9,25 @@
         <field name="arch" type="xml">
             <search>
                 <field name="name" />
+                <field name="origin" />
                 <field name="user_id" />
                 <field name="tag_ids" />
+                <filter
```
Comment (pedrobaeza) [inline rma/views/rma_views.xml:15]: Shouldn't a separator be here?

Comment (rousseldenis) [reply rma/views/rma_views.xml:15]: @sbejaoui

Comment (sbejaoui) [reply rma/views/rma_views.xml:15]: @pedrobaeza , no need for a separator between the search fields and filters

image

**Review by rousseldenis — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**